### ISSUE TITLE
TextEdit edited callback doesn't trigger

### ISSUE THREAD

**Issue by sigmaSd** [slint-ui/slint#6245]
```slint
import { TextEdit, LineEdit } from "std-widgets.slint";
export component App inherits Window {
    height: 400px;
    width: 400px;
    VerticalLayout {

        a := Text {
            text: "Hello World";
        }

        TextEdit {
            edited(text) => {
                a.text = text;
            }
        }
    }
}
```

if I change TextEdit to LineEdit it works

**Comment by ogoffart** [comment]
Thanks for filing an issue.
This was already reported in https://github.com/slint-ui/slint/issues/5848 and fixed in https://github.com/slint-ui/slint/pull/5850 which will be in Slint 1.8.
